projects/map-ol/src/lib/map-ol.service.ts

```diff
@@ -73,13 +73,27 @@ const TITLE_KEY = 'title';
 const WebMercator = 'EPSG:3857';
 const WGS84 = 'EPSG:4326';
 
+/**
+ * While @dlr-eoc/services-layers.popup already contains instructions about how to build a popup,
+ * IPopupArgs adds additional, map-ol-specific context like the olLayer, the feature, and the olMapBrowserEvent.
+ */
 export interface IPopupArgs {
   modelName: string;
-  properties: any;
+  properties: popup['properties']; // have been filtered beforehand by popup['filterkeys'] (if given)
```

Comment:
This comment is not correct because the properties get filtered/adjusted later.
See
- https://github.com/dlr-eoc/ukis-frontend-libraries/blob/angular-popup/projects/map-ol/src/lib/map-ol.service.ts#L1315
- https://github.com/dlr-eoc/ukis-frontend-libraries/blob/angular-popup/projects/map-ol/src/lib/map-ol.service.ts#L1320
- https://github.com/dlr-eoc/ukis-frontend-libraries/blob/angular-popup/projects/map-ol/src/lib/map-ol.service.ts#L1329
- https://github.com/dlr-eoc/ukis-frontend-libraries/blob/angular-popup/projects/map-ol/src/lib/map-ol.service.ts#L1339

then a new filtered object `popupProperties` gets passed to 
```ts
this.addPopup(args, popupProperties, null);

...

layerpopup.asyncPupup(popupProperties, (html) => {...

...

this.addPopup(args, popupProperties, null, layerpopup.event, layerpopup.single);
```

but we can remove this later on because it is only a command :)